py4DSTEM/process/phase/utils.py

```diff
@@ -413,7 +413,8 @@ def polar_coordinates(self, x, y):
         """Calculate a polar grid for a given Cartesian grid."""
         xp = self._xp
         alpha = xp.sqrt(x.reshape((-1, 1)) ** 2 + y.reshape((1, -1)) ** 2)
-        phi = xp.arctan2(x.reshape((-1, 1)), y.reshape((1, -1)))
+        # phi = xp.arctan2(x.reshape((-1, 1)), y.reshape((1, -1))) # bug in abtem-legacy and py4DSTEM<=0.14.9
+        phi = xp.arctan2(y.reshape((1, -1)), x.reshape((-1, 1)))
```

Comment:
This is fine as-is but I would point out that `.ravel()` does the same thing (and won't make a copy in this case)

Reply:
I thought `.ravel()` has the same copy-behaviour as `.reshape()`?  
This was originally copied from abtem (hence the origin of the pesky bug), but can however be simply broadcasted:
```python
alpha = xp.sqrt(x[:,None]**2 + y[None,:]**2)
phi = xp.arctan2(y[None,:], x[:,None])
```